db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
```

Comment:
db schema changes due to running `bundle exec rails db:migrate` due to the conflicts in the schema file, all changes were automatic